src/main/scala/vsys/blockchain/contract/DataEntry.scala

```diff
@@ -85,4 +87,17 @@ object DataEntry {
       case Left(l) => Left(l)
     }
   }
+
+  object ConvertHelper {
+
+    implicit def int2Byte(x:Int): Byte = x.toByte
+
+    implicit def dataEntry2Int(x: DataEntry): Int = Int32.deserializer(x.data)
+
+    implicit def dataEntry2BigInt(x: DataEntry): BigInt = BigInteger.deserializer(x.data)
+
+    implicit def dataEntry2Long(x: DataEntry): Long = Amount.deserializer(x.data)  // same as timestamp
```

Comment:
`Longs.fromByteArray` and `Ints.fromByteArray` can be used here directly

Reply:
yes, using `Longs.fromByteArray` and `Ints.fromByteArray` is better here.